web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/intro.md

```diff
@@ -26,36 +26,34 @@ backend.
 ## Creating A New Project With The Template
 
 There are two methods to creating a new project based on the
-`gatsby-wordpress-starter`:`
+`gatsby-wordpress-starter`:
 
-1. Clone the starter repo directly
-2. Use the `gatsby-cli`
+1. **Preferred Method** - Using our `create-pantheon-decoupled-kit`, or the
+   "CLI"
+2. Clone the starter repo directly
```

Comment:
I'm still not sold on this method being advertised at all.

Reply:
Can kind of see both arguments here, but the thing that tips me over the edge is that there are actually kind of three methods - the CLI, cloning the repo directly, and selecting the project in the dashboard (which technically clones the repo behind the scenes.) Also thinking about now Next, Astro etc present this, they rarely if ever point people to the repo directly.

So I'll side with only suggesting the CLI as well.